scripts/eurostat/regional_statistics_by_nuts/README.md

```diff
@@ -0,0 +1,53 @@
+# Importing EuroStat XXX Into Data Commons
+Author: <author>
+
+## Table of Contents
+
+1. [About the Dataset](#about-the-dataset)
+
+## About the Dataset
+
+### Download URL
+
+[TSV] file is available for download from <url>.
+
+### Overview
+The metadata for Population change - Demographic balance and crude rates at regional level (NUTS 3) is available online at [here](https://ec.europa.eu/eurostat/cache/metadata/en/demo_r_gind3_esms.htm)
+
+NUTS (Nomenclature of Territorial Units for Statistics) is a hierarchical classification for dividing regions of Europe. NUTSi, starting at NUTS1, is the ith classification which is more detailed than the NUTS(i-1). We’re importing population density for regions in the NUTS3 classification.
```

Comment:
I think we can just link to a page about NUTS after spelling out the acronym, and then list the P0/P1 datasets along with their covered geos (country to NUTS2, or country to NUTS3).